ci/test/test_integration

```diff
@@ -44,6 +44,9 @@ docker_compose_command up openssl
 
 echo "Starting Conjur..."
 docker_compose_command up -d conjur
+docker_compose_command exec -T conjur cp /oidc/certs/oidc-server.cert.pem /etc/ssl/certs/oidc-server.cert.pem
+docker-compose --file "$COMPOSE_FILE_PATH" --project-directory "${PWD}" \
```

Comment:
Couldn't use the short `docker_compose_command` for this command, because it break the individual words into separate quoted words which conflict with the `sh` command that expects a single quoted string